spec/index.html

```diff
@@ -674,8 +694,9 @@ <h2>Literals</h2>
       elements:</p>
 
     <ul>
-      <li>a <dfn>lexical form</dfn>, being a Unicode [[!UNICODE]] string,
-        which SHOULD be in Normal Form&nbsp;C [[!NFC]],</li>
+      <li>a <dfn>lexical form</dfn> consisting of a sequence of
+        <a data-lt="code point" class="lint-ignore">Unicode code points</a> [[!UNICODE]]
+        matching the <a data-cite="XML10#charsets">Char</a> production [[XML10]].</li>
```

Comment:
```suggestion
        matching the <a data-cite="XML11#charsets">Char</a> production [[XML11]].</li>
       Because RDF lexical forms match the Char production of XML they do not contain Unicode surrogate code points.
```